content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,86 @@ ifndef::include-only-once[]
 include::{root-path}_config.adoc[]
 endif::[]
 
-TODO: Add terms and definitions
+3D Asset::
+A 3D model. containing the 3D geometry, combined with assigned material properties and specific meta data.
```

Comment:
The combination of an object's 3D geometry, assigned material properties, and specific metadata.

Reply:
done